tools/swift-test-utils/SwiftTestUtils.swift

```diff
@@ -1,24 +1,24 @@
 import Foundation
 
-struct CommandLineArguments {
-  struct MissingArgumentError: LocalizedError {
+public struct CommandLineArguments {
+  fileprivate struct MissingArgumentError: LocalizedError {
     let argName: String
 
     var errorDescription: String? {
       return "Missing required argument: \(argName)"
     }
   }
-  struct UnkeyedArgumentError: LocalizedError {
+  fileprivate struct UnkeyedArgumentError: LocalizedError {
```

Comment:
IMHO this should be `public` so that you can specifically catch the error. Same goes for `MissingArgumentError`.

Reply:
Ok, will update!